src/App.scss

```diff
@@ -52,11 +50,15 @@
   position: fixed !important;
 }
 
-@media screen and (min-width: 1255px) {
+@media screen and (min-width: 768px) {
```

Comment:
There's a mixin we can use here that keeps this nice and clean:

You have to import these three files to be able to use it:

```
@import "bootstrap/scss/functions"; 
@import "bootstrap/scss/variables";
@import "bootstrap/scss/mixins";
```

Once those are there you can use:

```
@include media-breakpoint-up(sm) {
  // Put your styles here
}
```

\- or -

```
@include media-breakpoint-up(lg) {
  // Put your styles here
}
```

If we're going to overriding a lot of things per breakpoint in this file, I would recommend switching to this pattern for clarity.